Codeinwp/otter-blocks
# Use SDK function for upgrade links and survey

Closes #.


### Summary


- Add lang code to upgrade links.
- Use Formbricks from SDK, which enhances the payload with the lang code

### Test instructions


- In the dashboard, check if `survey_deps` is loaded into the page.
- Check if the upgrade links have `lang=de` when WP is set to German.



---- 

### Checklist before the final review
